client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -238,7 +238,7 @@ class DomainRegistrationSuggestion extends React.Component {
 
 		let title, progressBarProps;
 		if ( isRecommended ) {
-			title = translate( 'Best Match' );
+			title = this.props.showTestCopy ? 'Our recommendation' : translate( 'Best Match' );
```

Comment:
Both are on purpose. Since this A/B test is for `en` only, I thought it would be better to see if the test is a success before submitting these strings for translation(which would incur us costs). 

On the capitalization, though I just followed the copy recommendation,  I will double confirm with those handling the copy changes. Thanks for highlighting.

Reply:
The brand book says we should capitalize titles and buttons.

Reply:
Agreed. I am going to go ahead and merge with the capitalized text.